docs/docs/30-how-to-guides/30-docker-image.md

```diff
@@ -17,7 +17,9 @@ easiest option for experimenting locally with Kargo Render!
 Example usage:
 
 ```shell
-docker run -it ghcr.io/akuity/kargo-render:v0.1.0-rc.36 \
+docker run -it ghcr.io/akuity/kargo-render:v0.1.0-rc.35 \
```

Comment:
I suspect that we'll see `.36` exist soon, as this was just changed to that in [#261](https://github.com/akuity/kargo-render/pull/261/files#diff-c044d06f2ec0d0f3ea834ffd959a51c14b6ec93f1e8ce98c269311e9e4b76487) a couple of hours ago. But @krancour could confirm.